packages/next/src/server/config.ts

```diff
@@ -686,39 +686,39 @@ function assignDefaults(
-          'modularize-import-loader?name={{ member }}&from=default&as=default&join=./icons/braces!lucide-react',
+          'modularize-import-loader?name={{ member }}&from=default&as=default&join=../esm/icons/braces!lucide-react',
         '(CircleSlashed|LucideCircleSlashed|CircleSlashedIcon)':
-          'modularize-import-loader?name={{ member }}&from=default&as=default&join=./icons/circle-slash-2!lucide-react',
+          'modularize-import-loader?name={{ member }}&from=default&as=default&join=../esm/icons/circle-slash-2!lucide-react',
         '(SquareGantt|LucideSquareGantt|SquareGanttIcon)':
-          'modularize-import-loader?name={{ member }}&from=default&as=default&join=./icons/gantt-chart-square!lucide-react',
+          'modularize-import-loader?name={{ member }}&from=default&as=default&join=../esm/icons/gantt-chart-square!lucide-react',
         '(SquareKanbanDashed|LucideSquareKanbanDashed|SquareKanbanDashedIcon)':
-          'modularize-import-loader?name={{ member }}&from=default&as=default&join=./icons/kanban-square-dashed!lucide-react',
+          'modularize-import-loader?name={{ member }}&from=default&as=default&join=../esm/icons/kanban-square-dashed!lucide-react',
         '(SquareKanban|LucideSquareKanban|SquareKanbanIcon)':
-          'modularize-import-loader?name={{ member }}&from=default&as=default&join=./icons/kanban-square!lucide-react',
+          'modularize-import-loader?name={{ member }}&from=default&as=default&join=../esm/icons/kanban-square!lucide-react',
         '(Edit3|LucideEdit3|Edit3Icon)':
-          'modularize-import-loader?name={{ member }}&from=default&as=default&join=./icons/pen-line!lucide-react',
+          'modularize-import-loader?name={{ member }}&from=default&as=default&join=../esm/icons/pen-line!lucide-react',
         '(Edit|LucideEdit|EditIcon|PenBox|LucidePenBox|PenBoxIcon)':
-          'modularize-import-loader?name={{ member }}&from=default&as=default&join=./icons/pen-square!lucide-react',
+          'modularize-import-loader?name={{ member }}&from=default&as=default&join=../esm/icons/pen-square!lucide-react',
         '(Edit2|LucideEdit2|Edit2Icon)':
-          'modularize-import-loader?name={{ member }}&from=default&as=default&join=./icons/pen!lucide-react',
+          'modularize-import-loader?name={{ member }}&from=default&as=default&join=../esm/icons/pen!lucide-react',
         '(Stars|LucideStars|StarsIcon)':
-          'modularize-import-loader?name={{ member }}&from=default&as=default&join=./icons/sparkles!lucide-react',
+          'modularize-import-loader?name={{ member }}&from=default&as=default&join=../esm/icons/sparkles!lucide-react',
         '(TextSelection|LucideTextSelection|TextSelectionIcon)':
-          'modularize-import-loader?name={{ member }}&from=default&as=default&join=./icons/text-select!lucide-react',
+          'modularize-import-loader?name={{ member }}&from=default&as=default&join=../esm/icons/text-select!lucide-react',
```

Comment:
IMO it would be best if we just delete all those lines.
https://github.com/lucide-icons/lucide/pull/1486

Reply:
@jguddas Thank for your PR, this looks like best solution for all those aliases.

@shuding I will create another PR with those aliases lines removed after we released new version of lucide with the changes from @jguddas in https://github.com/lucide-icons/lucide/pull/1486